npm/mount-utils/src/index.ts

```diff
@@ -1,42 +1,3 @@
-/**
```

Comment:
I am going to miss this a lot. I think we should reconsider deprecating this.

If we do, I think we should warn and **not** throw - I use this API heavily in my tests, and I think a lot other people do, too. Either way, it'd be nice to be able to upgrade to v11 without making major changes to my tests - if we could simply warn and deprecate in v12, that'd be nice, at least from my point of view.

Reply:
Edit, talked in slack, we will deprecate but we need a good blog post or docs explaining "here's how you should do it now, and this is why it's better".

Reply:
done https://github.com/cypress-io/cypress/pull/24419/commits/e8af254b28368c07e270dab9c6ea0b7a92e12387